### PR TITLE
Add ISM support

### PR DESCRIPTION
**Pull request by valnoel**

### CODE REVIEW

File: src/ism/manifest.rs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+}
+
+#[derive(Debug, YaDeserialize, YaSerialize)]
+struct Head {
+  meta: Vec<Meta>,
+  #[yaserde(rename = "paramGroup")]
+  param_group: Vec<ParamGroup>,
+}
+
+impl Default for Head {
```
Comment (MarcAntoine-Arnaud) [inline src/ism/manifest.rs]: you can add this default to `derive` directly

File: src/main.rs [hunk 2]
```diff
@@ -3,36 +3,63 @@ extern crate amqp_worker;
...
-  start_worker(&DASH_MANIFEST_EVENT);
+  match env::var("MANIFEST_MODE")
+    .unwrap_or(DASH.to_string())
+    .as_str()
+  {
+    ISM => {
+      info!("Start worker with ISM mode...");
+      start_worker(&ISM_MANIFEST_EVENT)
+    }
+    _ => {
```
Comment (MarcAntoine-Arnaud) [inline src/main.rs]: is it possible to check DASH here and not starting worker if it's something else (better in case of a wrong spelling..)

**Review by MarcAntoine-Arnaud — CHANGES_REQUESTED** [review]

File: src/main.rs (hunk 2 above)
Comment (valnoel) [reply src/main.rs]: See 3177fb5

File: src/ism/manifest.rs (hunk 1 above)
Comment (valnoel) [reply src/ism/manifest.rs]: See 2578ac7

**Review by MarcAntoine-Arnaud — APPROVED** [review]

**Pull request merged**